front/app/resources/GetUsers.tsx

```diff
@@ -118,6 +118,11 @@ export default class GetUsers extends React.Component<Props, State> {
           })
         )
         .subscribe(({ users, queryParameters }) => {
+          const currentPageNumberFromURL = getPageNumberFromUrl(
```

Comment:
Here we don't do isNilOrError, but just working with the types as they exist. users apparently can't be null here. :-)